Lib/keyword.py

```diff
@@ -1,98 +1,56 @@
 #! /usr/bin/env python3
```

Comment:
This file is no longer executable, so it does not need a shebang.

Remove also the executable bit.

Reply:
Done in c17c209